lighty-core/lighty-codecs-util/src/test/java/io/lighty/codecs/util/AbstractCodecTest.java

```diff
@@ -40,10 +40,10 @@
 import org.opendaylight.yangtools.yang.data.api.YangInstanceIdentifier.NodeIdentifierWithPredicates;
 import org.opendaylight.yangtools.yang.data.api.schema.MapEntryNode;
 import org.opendaylight.yangtools.yang.data.api.schema.NormalizedNode;
-import org.opendaylight.yangtools.yang.data.impl.schema.builder.impl.ImmutableContainerNodeBuilder;
-import org.opendaylight.yangtools.yang.data.impl.schema.builder.impl.ImmutableLeafNodeBuilder;
-import org.opendaylight.yangtools.yang.data.impl.schema.builder.impl.ImmutableMapEntryNodeBuilder;
-import org.opendaylight.yangtools.yang.data.impl.schema.builder.impl.ImmutableMapNodeBuilder;
+import org.opendaylight.yangtools.yang.data.spi.node.ImmutableNodes;
+import org.opendaylight.yangtools.yang.data.spi.node.impl.ImmutableContainerNodeBuilder;
```

Comment:
IMO these should be hidden - use `ImmutableNodes` only.

Reply:
resolved in:
https://github.com/PANTHEONtech/lighty/commit/91c849dc9f9d9a1ce8e09c37fb85ecf5762c2212

Reply:
Not completed yet.

Reply:
Still we have lot of import org.opendaylight.yangtools.yang.data.spi.node.impl.* imports, even deprecated Builders.
See: https://github.com/opendaylight/yangtools/commit/1600e4a57127dc33178bb2953402495d440c8578 for explanation.

Reply:
Removed all builders and deprecated immutable builders:
https://github.com/PANTHEONtech/lighty/pull/1860/commits/9ff9a255759a05e7a443ff611b8a176b1685b126